src/main/java/com/hivemq/cli/mqtt/MqttClientExecutor.java

```diff
@@ -331,13 +257,13 @@ else if (publish.isDebug()) {
 
                             TypeSwitch.when(publishResult)
                                     .is(MqttPublishResult.MqttQos1Result.class, qos1Result -> {
-                                        Logger.debug("received PUBACK: '{}' for PUBLISH to Topic:  {}", trimMessage(p), topic);
+                                        Logger.debug("received PUBACK: '{}' for PUBLISH to Topic: {}", trimMessage(p), topic);
                                     })
                                     .is(MqttPublishResult.MqttQos2Result.class, qos2Result -> {
-                                        Logger.debug("received PUBREC: '{}' for PUBLISH to Topic:  {}", trimMessage(p), topic);
+                                        Logger.debug("received PUBREC: '{}' for PUBLISH to Topic: {}", trimMessage(p), topic);
                                     })
                                     .is(MqttPublishResult.class, qos0Result -> {
-                                        Logger.debug("acknowledged PUBLISH: '{}' for PUBLISH to Topic:  {}", trimMessage(p), topic);
+                                        Logger.debug("acknowledged PUBLISH: '{}' for PUBLISH to Topic: {}", trimMessage(p), topic);
```

Comment:
Are this the correct responses? PUBACK, PUBREC, acknowlegded PUBLISH?

Reply:
QOS 0 you don't get any acknowledge
QOS 1 you get PUBACK
QOS 2 you get first PUBREC then the client sends PUBREL finally the broker sends PUBCOMP